charts/gardener/controlplane/charts/application/templates/rbac-system.yaml

```diff
@@ -108,4 +163,67 @@ subjects:
   - kind: ServiceAccount
     name: "{{ required ".Values.global.scheduler.serviceAccountName is required" .Values.global.scheduler.serviceAccountName }}"
     namespace: garden
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+###
+# RBAC roles for bootstrapping gardenlets
+###
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:seeds
+rules:
```

Comment:
Probably not, we plan to open a separate follow-up issue for this, see above PR description:

> Write seed authenticator that only allows the Gardenlet to handle resources related to its targeted seed(s)

In this first version it will simply have privileges for everything, just like the GCM today. :)